server/src/main/java/org/tctalent/server/api/admin/UserAdminApi.java

```diff
@@ -155,6 +178,9 @@ private DtoBuilder userDto() {
                 .add("usingMfa")
                 .add("mfaConfigured")
                 .add("partner", PartnerDtoHelper.getPartnerDto())
+                .add("emailVerified")
+                .add("emailVerificationToken")
```

Comment:
I don't understand why the Angular code needs to see or process the token or its issuedDate. All that can be managed by the server.

Reply:
I initially didn’t send them, but later used them in a function to check if the user sent an email and if the token expired. This triggers the modal to say the email was already sent. However, there could be a better frontend approach, I will check.

Reply:
I removed the token and its issuedDate.